### PR TITLE
manual rebase 2023-07-06

### PR DESCRIPTION
**Pull request by dhellmann**

### CODE REVIEW

**Comment by dhellmann** [comment]
The first commit in this series is part of #2005 and we should merge that separately, then squash the commits in this PR when it merges.

File: assets/components/csi-snapshot-controller/role.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+kind: Role
```
Comment (copejon) [inline assets/components/csi-snapshot-controller/role.yaml:1]: Leader election is disabled on the csi snapshotter. This being a manual rebase, let's go ahead and drop these (role / binding). I'm looking at the script now to see what's causing them to populate, but they shouldn't be included.

**Comment by copejon** [comment]
> @copejon I think your comment [#2003 (comment)](https://github.com/openshift/microshift/pull/2003#issuecomment-1623881019) applies here, too.
> 
> Can you look at whatever we need to change in the rebase script to make that happen? Perhaps we should not merge this PR at all?

Looking at it now.  Shouldn't be a huge issue. I'll have a PR up in a bit to fix it, then we can retry the rebase.

**Comment by dhellmann** [comment]
/close

obsoleted by #2008

**Pull request closed without merging**